runtime/sema/void_type.go

```diff
@@ -28,7 +28,7 @@ var VoidType = &SimpleType{
 	IsInvalid:            false,
 	IsResource:           false,
 	Storable:             false,
-	Equatable:            false,
+	Equatable:            true,
```

Comment:
Nice, lucky that `VoidValue` already implements `EquatableValue` 👍

Reply:
yeah, it's the same story as the Paths. just need to turn the feature on :)